src/families/tron/specs.js

```diff
@@ -0,0 +1,152 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import expect from "expect";
+import type { Transaction } from "./types";
+import { getCryptoCurrencyById, parseCurrencyUnit } from "../../currencies";
+import { pickSiblings } from "../../bot/specs";
+import type { AppSpec } from "../../bot/types";
+import deviceActions from "./speculos-deviceActions";
+import { NetworkDown } from "@ledgerhq/errors";
+
+const currency = getCryptoCurrencyById("tron");
+const minimalAmount = parseCurrencyUnit(currency.units[0], "10");
+
+const tron: AppSpec<Transaction> = {
+  name: "Tron",
+  currency,
+  appQuery: {
+    model: "nanoS",
+    appName: "Tron",
+    appVersion: "0.2.x",
+  },
+  mutations: [
+    {
+      name: "move 50% to another account",
+      transaction: ({ account, siblings, bridge }) => {
+        invariant(account.balance.gt(minimalAmount), "balance is too low");
```

Comment:
most usage of `account.balance` shall be replaced by `maxSpendable` because not all your balance is spendable.